src/routes/log.ts

```diff
@@ -371,6 +371,12 @@ router.route('/stopRPCCapture').get(async function (req: Request, res: Response)
   CONFIG.statLog = false
   res.json({ message: 'RPC interface recording disabled' }).status(200)
 })
+router.route('/countRPCCapture').get(async function (req: Request, res: Response) {
+  if (!CONFIG.statLog) {
+    return res.json({ message: 'Interface stats recording disabled' }).status(304)
```

Comment:
304 status code is really intended to indicate the resource hasn't changed generally around caching. 
I think 200 OK or 204 No Content would make more sense